### DISCUSSION TITLE
In case it helps: uploding AMR files.

### DISCUSSION THREAD

**Discussion by thesamim** [vzhd1701/enex2notion#14] [General]
First of all, thanks to @vzhd1701 for creating enex2notion. VERY helpful.

This may apply to other attachments, have only run into it with an AMR recording.

If you get a "requests.exceptions.HTTPError: File does not have an extension." while uploading, find the offending the file, find the resource attributes, add an extension.
So:
```
      <resource-attributes>
        <timestamp>20140113T132144Z</timestamp>
        <file-name>Jan 13, 2014 7:21:44 AM</file-name>
        <attachment>false</attachment>
      </resource-attributes>
```
becomes:
```
      <resource-attributes>
        <timestamp>20140113T132144Z</timestamp>
        <file-name>Jan 13, 2014 7:21:44 AM.amr</file-name>
        <attachment>false</attachment>
      </resource-attributes>
```

**Comment by vzhd1701** [comment]
Thank you for reporting this issue. I fixed it in the new version 0.2.5.
